File: src/main/java/com/application/vendetta/entities/Comments.java
package com.application.vendetta.entities;

import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
@Entity
@Table(name = "comments")
public class Comments {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;
    private String text;
    private String emailOfCreator;
    @ManyToOne(fetch = FetchType.LAZY)
    private Markers marker;
}

File: src/frontend/src/app/models/comments.ts
import { Markers } from "./markers";

export class Comments{
    constructor(
        private text: string,
        private emailOfCreator:string,
        private marker:Markers,
        private id?:number
        
    ){}

    getText(){
        return this.text;
    }

    getEmailOfCreator(){
        return this.emailOfCreator;
    }
}